Gemfile

```diff
@@ -1,4 +1,5 @@
 source "https://rubygems.org" do
   gem "jekyll"
   gem "github-pages", group: :jekyll_plugins
+  gem "jekyll-github-metadata"
```

Comment:
remove the need for the hard coded `github.repository_url` in the `_config.yaml`. Hooray for one less point of configuration!